glossary.md

```diff
@@ -317,7 +317,7 @@ Nếu bạn cho rằng một từ không nên dịch ra tiếng Việt, bạn c
 | penalty                            | lượng phạt                 | [https://git.io/JvQAP](https://git.io/JvQAP) |
 | perceptron                         | perceptron                 | [https://git.io/JvohC](https://git.io/JvohC) |
 | performance                        | chất lượng                 | [https://git.io/JvQx4](https://git.io/JvQx4) |
-| perplexity (metric)                | độ rối rắm                 |  |
+| perplexity (metric)                | độ hỗn độn                 |  |
```

Comment:
```suggestion
| perplexity (metric)                | độ rối                 |  |
```

Reply:
Em đưa thêm một số ví dụ tìm được trên Google cho thấy perplexity (PP) có thể coi là entropy trong NLP (độ hỗn độn trong ngữ cảnh NLP). Trong link, họ còn dẫn ra các slide từ bài giảng NLP của Stanford và các bài báo.

Độ rối và độ lủng củng em cảm nhận là thước đo về "độ không tốt" của câu @ngcthuong  @cuongvng . Tuy nhiên, theo nguồn Wikipedia (mang tính hàn lâm đôi chút tuy không thấy ghi rõ nguồn) và cũng từ công thức perplexity, sẽ gặp trường hợp một câu có giá trị PP cao dù câu đó có hay tới đâu vì xác suất gặp trong lúc huấn luyện là rất thấp.

Có thể đó cũng là lý do mà các thầy ở BK và trường khác không sửa lại từ đó trong bài tập lớn cũng như luận văn của sv.

> Intuitively, perplexity can be understood as a measure of uncertainty.
https://thegradient.pub/understanding-evaluation-metrics-for-language-models/

> The perplexity is the exponentiation of the entropy
https://en.wikipedia.org/wiki/Perplexity

> you can safely think of the concept of perplexity as entropy
https://towardsdatascience.com/perplexity-intuition-and-derivation-105dd481c8f3

Reply:
Em vote độ hỗn độn vì nếu có hiểu nhầm `entropy` thì cũng hợp lý.

Reply:
Qua Google thì em thấy là `entropy` có thể được giữ nguyên, hoặc được dịch định nghĩa là `độ hỗn độn` (thi thoảng thấy `độ hỗn loạn`) trong các giáo trình vật lý, hóa học, thiên văn học, lý thuyết thông tin, v.v.
Entropy có thể được giữ nguyên, perplexity nếu được dịch sang tiếng Việt thì hay hơn vì đọc cảm giác hơi dài (perplexity của câu A).